tests/test_regrtest.py

```diff
@@ -248,7 +248,7 @@ def with_metaclass(meta, *bases):
             class metaclass(meta):
                 def __new__(cls, name, this_bases, d):
                     return meta(name, bases, d)
-        return type.__new__(metaclass, 'temporary_class', (), {})
+            return type.__new__(metaclass, 'temporary_class', (), {})
```

Comment:
Hopefully the crash referred to in the bitbucket issue that has been lost to time didn't depend on this syntax error. 🙏

Reply:
From my understanding, the only part that mattered was the «six.with_metaclass» in the line below. The piece of code above could have been either simplified significantly or omitted altogether

Reply:
Ugh. Thanks for looking into it.